library/build.gradle

```diff
@@ -22,7 +22,6 @@ android {
         main {
             java {
                 srcDirs = [
-                        'src/main/java',
                         'src/main/android'
```

Comment:
Can we move the entire source to`src/main/java` now?

Reply:
thx 🙇3126812